modules/admin_manual/pages/configuration/server/oc_server_tuning.adoc

```diff
@@ -123,8 +123,7 @@ ownCloud documentation. However, here are three links that can help you
 find further information:
 
 * http://de.slideshare.net/PGExperts/five-steps-perform2013[Five Steps to PostgreSQL Performance]
-* http://grokbase.com/t/postgresql/pgsql-admin/103qcpdrpf/tuning-auto-vacuum-for-highly-active-tables#20100323hfs3jtjuaywwufukoqtexkpjti[Tuning
-the autovacuum proceff for tables with huge update workloads (oc_filecache)]
+* https://grokbase.com/t/postgresql/pgsql-admin/103qcpdrpf/tuning-auto-vacuum-for-highly-active-tables#20100323hfs3jtjuaywwufukoqtexkpjti[Tuning the autovacuum proceff for tables with huge update workloads (oc_filecache)]
```

Comment:
```suggestion
* https://grokbase.com/t/postgresql/pgsql-admin/103qcpdrpf/tuning-auto-vacuum-for-highly-active-tables#20100323hfs3jtjuaywwufukoqtexkpjti[Tuning the autovacuum process for tables with huge update workloads (oc_filecache)]
```

Reply:
That site seems to be "unmaintained" or???

----------
grokbase.com uses an invalid security certificate.
 
The certificate is not trusted because the issuer certificate has expired.
 
Error code: SEC_ERROR_EXPIRED_ISSUER_CERTIFICATE

----------
Certificate expired Fri, 22 Jan 2021 10:41:47 GMT

Reply:
I know. If you have a replacement suggestion or another solution, I am happy to implement.

Reply:
Let's leave it like this for now. The purpose of this PR is just to get links pointing somewhere "reasonable".